deploy/charts/cert-manager/values.yaml

```diff
@@ -284,7 +284,7 @@ topologySpreadConstraints: []
 # controller-manager. See:
 # https://github.com/kubernetes/kubernetes/blob/806b30170c61a38fedd54cc9ede4cd6275a1ad3b/cmd/kubeadm/app/util/staticpod/utils.go#L241-L245
 livenessProbe:
-  enabled: false
+  enabled: true
```

Comment:
The comment is out-of-date, I think the liveness probe becomes more useful once we add the clock skew check. As far as I understand, the probe was disabled because it was expected to never fail (when it was only checking the leader election).